src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Replace the hardcoded value of '3' with a constant that explains what this value represents, such as 'SHORT_TERM_RENTAL_DAYS'. Similarly, replace '20' with a constant like 'SHORT_TERM_DISCOUNT'.